### PR TITLE
PLG-111: Track Page View Event

### PR DESCRIPTION
**Pull request by nedick007**
New class providing necessary data for tracking page views.

### CODE REVIEW

**Comment by nedick007** [comment]
@adamof ready for review

File: Block/Analytics.php [hunk 1]
```diff
@@ -14,15 +14,19 @@ public function __construct(
         \Magento\Framework\App\Action\Context $actionContext,
         \Metrilo\Analytics\Helper\Data        $helper,
         \Magento\Framework\Registry           $registry,
+        \Magento\Framework\View\Page\Title    $pageTitle,
+        \Magento\Framework\UrlInterface       $urlInterface,
```
Comment (adamof) [inline Block/Analytics.php]: What happened to the idea that the logic for each event should be contained in the class for the particular event?

Comment (nedick007) [reply Block/Analytics.php]: This is result of using a wrong pattern for instantiating the class. Will fix it using the proxy pattern.

**Comment by nedick007** [comment]
@adamof ready for review

**Comment by nedick007** [comment]
Tested on QA env.

**Pull request merged**